src/Lidarr.Api.V1/Config/MetadataProviderConfigModule.cs

```diff
@@ -11,7 +13,7 @@ public class MetadataProviderConfigModule : LidarrConfigModule<MetadataProviderC
         public MetadataProviderConfigModule(IConfigService configService)
             : base(configService)
         {
-            SharedValidator.RuleFor(c => c.MetadataSource).IsValidUrl();
+            SharedValidator.RuleFor(c => c.MetadataSource).IsValidUrl().When(c => !c.MetadataSource.Equals(String.Empty));
```

Comment:
Should use `!c.MetadataSource.IsNullOrWhiteSpace()` here to cover either case?

Reply:
True I originally did that .. but do you want to allow all the spaces.. or just want to allow a blank field?

Reply:
Functional I don’t think it makes a difference either way as we check for null or space before setting, whichever is easier on user and less error prone.